frontend/src/components/single-contract/StartJobButton.tsx

```diff
@@ -34,7 +35,10 @@ export const StartJobButton: React.FC = () => {
 
         <StartJobModal
           jobId={data.id}
-          deposit={data.deposit}
+          deposit={multiplyByIntegerPercentage(
```

Comment:
This is just for display purposes right? Like we're calculating the actual percentage in the contract?

Reply:
This code is replaced in the next PR.